src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpAvailabilityCheckerTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.sendletter.services;
+
+import org.assertj.core.api.AbstractBooleanAssert;
+import org.junit.Test;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FtpAvailabilityCheckerTest {
+
+    private static AbstractBooleanAssert<?> assertThatFtpCheck(String fromHour, String toHour, String now) {
```

Comment:
It would be best to have this at the bottom, after test methods.

Reply:
Although private, it is a static method and should be easier to spot at the beginning of the class. I would keep it where it is, but could move if you really, really wanted. :)